flow/project.py

```diff
@@ -1812,6 +1812,10 @@ def submit(self, bundle_size=1, jobs=None, names=None, num=None, parallel=False,
         # Regular argument checks and expansion
         if jobs is None:
             jobs = self  # select all jobs
+        if isinstance(names, six.string_types):
+            raise ValueError(
+                "The 'names' argument must be a sequence of strings, however you "
+                "provided a single string.")
```

Comment:
LGTM, however, I think adding the string that the user provided could add some clarity? Something like  
```suggestion
                "provided a single string: {}.".format(names))
```
I think it could help a user see more clearly the input they are providing is wrong. 

~Also, is there a reason we say sequence?~  Never mind, that is what we call it in the documentation :) Yay consistency!